indico/modules/events/management/util.py

```diff
@@ -182,3 +183,35 @@ def _process_attachments(folders):
 
     else:
         raise TypeError('Unexpected object of type {}: {}'.format(type(root).__name__, root))
+
+
+def find_unregistered_users(person_list):
+    """Get a set of users which don't have an Indico account
+
+    :param person_list: List of persons to check
+    """
+    return {pl for pl in person_list if pl.person.user is None}
+
+
+def flash_if_unregistered(former_person_list, new_person_list, event):
+    """Flash a message when user(s) with no Indico account is added
+
+    :param former_person_list: List of persons before new users were added
+    :param new_person_list: List of persons with new users
+    :param event: Current event
+    """
+    added_non_users = len(find_unregistered_users(new_person_list) - former_person_list)
+    if added_non_users:
+        msg = _(' An Indico account may be needed to upload materials and/or manage contents. ')
```

Comment:
Don't put surrounding whitespace here. It makes translation ugly and translators are likely to remove it in their translations.